code/online_challenges/src/project_euler/problem_020/problem_020.py

```diff
@@ -0,0 +1,8 @@
+factorial = 1
+for i in range(100):
+	factorial *= (i + 1)
+sum = 0
```

Comment:
Add space below. Also, don't use sum for a name.

Reply:
Where should I add space?

Reply:
@arnavb Can you please suggest changes here?

Reply:
By space I meant newline. Don't use sum for a variable name.

Reply:
Thanks @arnavb